CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## TBD
+### Changed
+* vacuum-tool: Removed classes for `TablePathResolverTest` to use the ones from [hcommon-hive-metastore](https://github.com/HotelsDotCom/hcommon-hive-metastore).
```

Comment:
Since they were public we should go for a major version bump of this project.

Reply:
That's true, thanks :)

Reply:
Done